src/solver/hydro/management/management.cpp

```diff
@@ -351,6 +351,27 @@ bool HydroManagement::checkMinGeneration(uint numSpace)
     return ret;
 }
 
+void HydroManagement::prepareFinalReservoirLevelData(uint numSpace, uint year)
```

Comment:
What we mostly do here is modify the inflows due to forced final level, right ?
So why not call this function **changeInflowsDueToFinalLevels** ?

Reply:
`changeInflowsToAccomodateFinalLevels` ?